xml/jtag_registers.xml

```diff
@@ -26,7 +26,29 @@
         sdesc="For Debugging" define="1">
         The size of this register will remain constant in future versions so
         that a debugger can always determine the version of the DTM.
-        <field name="0" bits="31:18" access="R" reset="0" />
+        <field name="0" bits="31:21" access="R" reset="0" />
+        <field name="errinfo" bits="20:18" access="R" reset="0">
+            This field may provide additional detail about an error that
+            occurred when communicating with a DM. It is updated whenever
+            \FdtmDmiOp is updated by the hardware.
+
+            <value v="0" name="unknown">
+                There is no error to report, or no further information available
+                about the error.
+            </value>
+
+            <value v="1" name="dmipower">
+                The error occurred because the DMI interface is powered down.
+            </value>
+
+            <value v="2" name="dmpower">
+                The error occurred because the DM is powered down.
+            </value>
+
+            TODO: Other errors? Parity?
```

Comment:
Another useful code would be that an error was reported from the DM.  An example might be if a debugger attempted to access a non-existent DMI register, or if a DMI request or response was corrupted (e.g. parity error).

Reply:
Added. Thanks.